CHANGELOG

```diff
@@ -48,6 +48,7 @@
 * Shift4: Add gateway adapter [ali-hassan] #4415
 * Rapyd: Correctly add `billing_address` [naashton] #4465
 * Credorax: Update processor response messages [jcreiff] #4466
+* ShiftFour: change gateway identifier and refactor code [ajawadmirza] #4469
```

Comment:
Can you change the `ShiftFour` to `Shift4`, just so it's searchable.

Reply:
done, thank you 😊